mcserver/utils.py

```diff
@@ -18,3 +47,17 @@ def send_otp_challenge(user):
     device.generate_challenge()
     print("CHALLENGE SENT")
 
+
+def get_processing_trials_count():
+    from mcserver.models import Trial
+    return Trial.objects.filter(status='processing').count()
```

Comment:
@olehkorkh-planeks I think we should check status='stopped'. Processing means it is actually being processed by a machine, stopped means it is seating in the queue waiting to be processed.